model/src/main/proto/screens.proto

```diff
@@ -5,14 +5,154 @@ package model;
+
+  // Screen name value for the scenario when the pin password activity is visible to the user.
+  PIN_PASSWORD_ACTIVITY = 26;
+
+  // Screen name value for the scenario when the profile picture activity is visible to the user.
+  PROFILE_PICTURE_ACTIVITY = 27;
+
+  // Screen name value for the scenario when the profile progress activity is visible to the user.
+  PROFILE_PROGRESS_ACTIVITY = 28;
+
+  // Screen name value for the scenario when the resume lesson activity is visible to the user.
+  RESUME_LESSON_ACTIVITY = 29;
+
+  // Screen name value for the scenario when the profile edit activity is visible to the user.
+  PROFILE_EDIT_ACTIVITY = 30;
+
+  // Screen name value for the scenario when the profile reset pin activity is visible to the user.
+  PROFILE_RESET_PIN_ACTIVITY = 31;
+
+  // Screen name value for the scenario when the profile rename activity is visible to the user.
+  PROFILE_RENAME_ACTIVITY = 32;
+
+  // Screen name value for the scenario when the profile list activity is visible to the user.
+  PROFILE_LIST_ACTIVITY = 33;
+
+  // Screen name value for the scenario when the story activity is visible to the user.
+  STORY_ACTIVITY = 34;
+
+  // Screen name value for the scenario when the topic activity is visible to the user.
+  TOPIC_ACTIVITY = 35;
+
+  // Screen name value for the scenario when the revision card activity is visible to the user.
+  REVISION_CARD_ACTIVITY = 36;
+
+  // Screen name value for the scenario when the question player activity is visible to the user.
+  QUESTION_PLAYER_ACTIVITY = 37;
+
+  // Screen name value for the scenario when the walkthrough activity is visible to the user.
+  WALKTHROUGH_ACTIVITY = 38;
+
+  // Screen name value for the scenario when the developer options activity is visible to the user.
+  DEVELOPER_OPTIONS_ACTIVITY = 39;
+
+  // Screen name value for the scenario when the view event logs activity is visible to the user.
+  VIEW_EVENT_LOGS_ACTIVITY = 40;
+
+  // Screen name value for the scenario when the mark topics completed activity is visible to the user.
+  MARK_TOPICS_COMPLETED_ACTIVITY = 41;
+
+  // Screen name value for the scenario when the math expression parser activity is visible to the user.
+  MATH_EXPRESSION_PARSER_ACTIVITY = 42;
+
+  // Screen name value for the scenario when the mark chapters completed activity is visible to the user.
+  MARK_CHAPTERS_COMPLETED_ACTIVITY = 43;
+
+  // Screen name value for the scenario when the mark stories completed activity is visible to the user.
+  MARK_STORIES_COMPLETED_ACTIVITY = 44;
+
+  // Screen name value for the scenario when the force network type activity is visible to the user.
+  FORCE_NETWORK_TYPE_ACTIVITY = 45;
+
+  // Screen name value for the scenario when the admin pin activity is visible to the user.
+  ADMIN_PIN_ACTIVITY = 46;
 }
 
+// Defines the current visible ui screen of the application.
 message CurrentAppScreen {
+  //S creen name value depicting the name of the currently visible ui screen.
```

Comment:
Space is in the wrong place--please fix.

Reply:
done.